app/app.config.js

```diff
@@ -1,4 +1,4 @@
-function routeConfig($stateProvider) {
+function config($stateProvider, $translateProvider, $translatePartialLoaderProvider) {

```

Comment:
I also can be arrow function, it is better to use `ngInject` in this case. You forger to add translatePartialLoaderProvider within .$inject